### ISSUE TITLE
Single Player crash

### ISSUE THREAD

**Issue by Kueder** [anegostudios/VintageStory-Issues#1200]
**Game Version:** 1.15.1  
**Platform:** Windows
**Modded:** No 

### Description
Single Player fails to start.

### How to reproduce
1. Install the game
2. Run it and create a single player world
3. It crashes when starting local server

### Expected behavior

Run the game

### Screenshots
[server-event.txt](https://github.com/anegostudios/VintageStory-Issues/files/6840620/server-event.txt)
[server-main.txt](https://github.com/anegostudios/VintageStory-Issues/files/6840628/server-main.txt)

### Logs
```
Attached Log
```

**Comment by Craluminum2413** [comment]
What mods are you using?

**Comment by Kueder** [comment]
None, fresh install

Sent from Mail for Windows 10

**Comment by Kueder** [comment]
Hi,

I had 1.15.1 RC2 installed before, but uninstalled it before installing 1.15.1.

I think something went wrong with the uninstall of RC2  but I cannot see where.

Maybe that info helps

Cheers,

Glenn

Sent from Mail for Windows 10

**Comment by copygirl** [comment]
Have you run the updater, or the full installer? I believe there's currently an issue with the updater where it doesn't delete files that have been removed from the game, or simply moved to another folder. Could you try to run the full installer? If that doesn't work, you could delete the folder `J:\Games\Vintagestory\assets` and reinstall again.

**Comment by Kueder** [comment]
No I have run the full install.
I have already removed manually after uninstall all vintagestory folders from my J drive and %appdata% roaming path

Then tried to reinstall but it is not working either.

I can give it a 5th try 😊


Sent from Mail for Windows 10

**Comment by Kueder** [comment]
Hey,

It finally works …. After doing the same thing manually I did a few times before ….

Well anyways, thanks for the quick investigation, it still is weird though.

So I uninstalled it via windows app, deleted manually the vintagestory folder on my J drive and deleted the %appdata%/roaming/VintageStoryData folder.

Cheers,

Do I need to close the issue ?

Glenn

Sent from Mail for Windows 10

**Comment by copygirl** [comment]
I'm glad you got it working. Not sure what was happening. In your `server-main.txt` the game said it only loaded 3 mods (the ones the game ships with), so I am unsure what it could have been complaining about. I will close the issue as resolved.

**Comment by Kueder** [comment]
Thank you 😊

I’m so happy it works again. It looks up for 2 paths for mods. For me my J drive and %appdata% /vintageStory/Mods in which I suspected there was something fishy but it was empty.

Maybe because I installed the RC2 in a different folder and something did not get deleted properly by the installer.

But I really deleted stuff manually, but maybe not the folder on my J drive …. Maybe that was the whole issue.
I did too many things trying to get it working again …. My bad.
Anyways, thanks a lot for your help 😊

Glenn

Sent from Mail for Windows 10
